tools/docker_build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+# A script to allow for easy local verification that docker containers are still working
+# TODO: Prevent it failing if a container with the same name already exists
```

Comment:
Would this script  be better served as a docker-compose file, or does it add anything over that?
E.g. something like the below would allow you to run:
`docker compose build test` to handle building, or `docker compose up test` to run it
```
services:
  prod:
    ...
  dev:
    ...
  test:
    ...
```